src/backend/BackendInterface.h

```diff
@@ -180,6 +180,21 @@ class BackendInterface
         std::uint32_t const ledgerSequence,
         boost::asio::yield_context& yield) const = 0;
 
+    // *** NFT methods
+    virtual std::optional<NFToken>
+    fetchNFToken(
+        ripple::uint256 const& tokenID,
+        std::uint32_t const ledgerSequence,
```

Comment:
Just FYI, the `const` part of `std::uint32_t const ledgerSequence` is ignored by the compiler in declarations, and this is a declaration.  There's little to no harm having the `const` present.  But it may not be doing what you think.  If this were the signature of a _definition_ then the `const` would have consequences.

No change required.  And it's a good thing that you're following the conventions set out by other declarations in this file.

Here's a link to an explanation of what's up: https://abseil.io/tips/109